source/render/ODFrameListener.cpp

```diff
@@ -92,8 +92,6 @@ ODFrameListener::ODFrameListener(Ogre::RenderWindow* renderWindow, Ogre::Overlay
 
     mGameMap = new GameMap(false);
 
-    mMiniMap = new MiniMap(mGameMap);
```

Comment:
Last time I checked it still was. Granted it was still way below the parts in ogre code.